EssentialCSharp.Web/Controllers/HomeController.cs

```diff
@@ -64,15 +69,36 @@ public IActionResult Announcements()
     {
         ViewBag.PageTitle = "Announcements";
         return View();
-    }
-
-    [Route("/home",
-        Name = "home")]
+    }
+
+    [Route("/home",
+    Name = "home")]
     public IActionResult Home()
     {
         return View();
     }
 
+    [HttpPost, Route("/home",
+        Name = "home")]
+    public async Task<IActionResult> Home(IFormCollection collection)
+    {
+        string hCaptchaSecret = _Configuration.GetValue<string>("HCaptcha:Secret") ?? throw new InvalidOperationException("HCaptcha:Secret is null");
+        string hCaptchaToken = collection["h-captcha-response"].ToString();
+        HttpResponseMessage response = await _CaptchaService.Verify(hCaptchaSecret, hCaptchaToken);
+        //if (response.IsSuccessStatusCode)
+        //{
+        //    _Logger.HomeControllerSuccessfulCaptchaResponse(Json(response));
+        //    return View();
+        //}
+        //else
+        //{
+        //    _Logger.HomeControllerSuccessfulCaptchaResponse(Json(response));
+
+        //    return RedirectToAction(nameof(Error), new { errorMessage = "Captcha Failed. Forbidden", statusCode = 403 });
+        //}

```

Comment:
Don't we need this for the captcha to work?

Reply:
Yes... but I don't think this is where we want the captcha exactly yet, so I was going to hold off on actually hooking it into the website until the future when we could discuss it and flush it out more. I wanted to keep the code though just for reference but I can also delete for this PR. Thoughts @keboo?

Reply:
That is fine. As long as we don't expect it working I am fine letting this merge.